pallets/automation-time/src/tests.rs

```diff
@@ -460,6 +526,24 @@ fn add_task_to_task_queue(
 	task_id
 }
 
+fn _add_transfer_task_to_task_queue(
```

Comment:
_ at the start

Reply:
also seems like you can combine most of this and add_notify_task_to_task_queue

Reply:
just removing for now, since we don't need